test/parallel/test-v8-features.js

```diff
@@ -0,0 +1,12 @@
+var common = require('../common');
+var assert = require('assert');
+var v8 = require('v8');
+
+eval('"use strict"; let x = 42');  // Should not throw.
+eval('"use strict"; const y = 42');  // Should not throw.
+assert.throws(function() { eval('"use strict"; class C {}'); });
+
+v8.setFlagsFromString('--harmony_classes');
+eval('"use strict"; class C {}');  // Should not throw.
+eval('"use strict"; let x = 42');  // Should not throw.
+eval('"use strict"; const y = 42');  // Should not throw.
```

Comment:
--harmony_classes implies --harmony_scoping.  The check is to ensure that the the latter is still on.  I believe let/const in sloppy mode is still under development.